ros/src/computing/planning/motion/packages/astar_planner/nodes/velocity_set/libvelocity_set.h

```diff
@@ -30,6 +30,7 @@ enum class EObstacleType
   ON_WAYPOINTS = 1,
   ON_CROSSWALK = 2,
   STOPLINE = 3,
+  ON_DECELERATE = 4,
```

Comment:
i somehow do not understand the meaning of this type of obstacle?

Reply:
exactly, I mean "obstacle on deceleration area on waypoints". but for now, we and you almost never use detection and planning for "deceleration", so I'll deprecate that.